src/frontend/src/features/blocknote/signature-block/index.tsx

```
@@ -0,0 +1,160 @@
+            });
+
+            if (isLoading) {
+                return <Spinner size="sm" />;
+            }
+
+            if (!preview?.html_body) {
+                return null;
+            }
+
+            return (
+                <div dangerouslySetInnerHTML={{ __html: preview.html_body }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)